shared/templates/rsyslog_logfiles_attributes_modify/oval.template

```diff
@@ -1,67 +1,67 @@
 <def-group oval_version="5.11">
-  <definition class="compliance" id="rsyslog_files_groupownership" version="1">
+  <definition class="compliance" id="rsyslog_files_{{{_RULE_ID }}}_ownership" version="1">
```

Comment:
Just `id="{{{_RULE_ID }}}"` should be enough here.

Reply:
Thanks :bow: done in [ba94e4c](https://github.com/ComplianceAsCode/content/pull/9789/commits/ba94e4ceee13613c8fc2527e66d9ea265ddc5690)